packages/relay/src/lib/hbarlimiter/index.ts

```diff
@@ -201,6 +219,44 @@ export default class HbarLimit {
     return this.reset;
   }
 
+  /**
+   * Estimates the total transaction fee in tinybars based on the call data size, file chunk size, and the current network exchange rate in cents.
+   *
+   * @param {number} callDataSize - The size of the call data in bytes.
+   * @param {number} fileChunkSize - The size of each file chunk in bytes.
+   * @param {number} currentNetworkExchangeRateInCents - The current exchange rate of HBAR to USD cents.
+   * @returns An object containing:
+   *   - `numFileCreateTxs` (number): The number of file creation transactions.
+   *   - `numFileAppendTxs` (number): The number of file append transactions.
+   *   - `totalFeeInTinyBar` (number): The estimated total transaction fee in tinybars.
+   */
+  estimateFileTransactionFee(
+    callDataSize: number,
+    fileChunkSize: number,
+    currentNetworkExchangeRateInCents: number,
```

Comment:
Please, can you pass `currentNetworkExchangeRate: IMirrorExchangeRate` object instead of `currentNetworkExchangeRateInCents: number` to this method and do the calculations inside?

```typescript
estimateFileTransactionFee(callDataSize: number, fileChunkSize: number, currentNetworkExchangeRate: IMirrorExchangeRate) {
  const exchangeRateInCents = currentNetworkExchangeRate.cent_equivalent / currentNetworkExchangeRate.hbar_equivalent;
  // rest of the method
}
```